recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,8 @@
+boost:
+  - 1.80.0
+boost_cpp:
+  - 1.80.0
```

Comment:
You should not need these keys since the global pinnings should provide them.

Reply:
The problem with doing that is [the global pinnings use 1.78.0]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/2ff674cb9adbdec9c3fff77b07ff9cec3734eb2c/recipe/conda_build_config.yaml#L274 ), but this feedstock is trying to use latest (currently 1.80.0). Dropping these results in falling back to 1.78.0, which doesn't look like what we want here.

Reply:
Wait who wants 1.80.0? The bot sent a PR since it found the version, but I don't see a specific user request for 1.80 (unless it is you @jakirkham!).

Reply:
This feedstock is in actuality an output of boost-cpp even if we want to manage it like a separate package. Having its version ahead of the global pinning doesn't seem like it is worth the effort to support.

Reply:
The feedstock started at 1.79.0 and was updated by the bot to 1.80.0 ( https://github.com/conda-forge/libboost-mpi-feedstock/pull/4 ).

There are users of conda-forge that stay mostly on the C++ side (and often have few dependencies). They are able to use the latest Boost without issues.

Reply:
Correct, this feedstock was outsourced from the boost-cpp to avoid adding complexité to the already complex feedstock, as discussed in conda-forge/boost-cpp-feedstock#116

Reply:
I am confused with the global pinning, shouldn't `boost` (aka the Boost.Python binding part) be pinned? Not `boost-cpp`?

Reply:
`boost-cpp` is what `boost` depends on. So that is what influences all of the Boost migration.

In any event agree with Matt we can drop these as long as we put this in a new branch ( https://github.com/regro/cf-scripts/issues/1569#issuecomment-1316972375 ). So have created a new `abi` branch and retargeted this.

```suggestion
```